docs/panel/js/pages/games/games.js

```diff
@@ -97,7 +99,8 @@ $(function () {
                                 cooldownTime = $('#cooldown-time'),
                                 gainPercent = $('#gain'),
                                 minBet = $('#min-bet'),
-                                maxBet = $('#max-bet');
+                                maxBet = $('#max-bet'),
+                                startPermission = helpers.getGroupIdByName($('#start-permission').find(':selected').text(), true);
```

Comment:
Never edit the remote panel. It is done via automation

Reply:
Reverted the change